### PR TITLE
Features

### PR DESCRIPTION
**Pull request by AsimKhan2019**
The stylesheet was still showing errors. Creating another pull request.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -164,16 +164,16 @@ <h1>About<br />Myself</h1>
...
                         </ul>
                     </li>
                     <li><b>Frameworks</b></li>
-                    <li><a href="#"><img src="img/seperator.svg" alt="seperator"></a></li>
+                    <!--<li><a href="#"><img src="img/seperator.svg" alt="seperator"></a></li>-->
                     <li><b>Skills</b></li>
-                    <li><a href="#"><img src="img/seperator.svg" alt="seperator"></a></li>
+                    <!--<li><a href="#"><img src="img/seperator.svg" alt="seperator"></a></li>-->
                 </ul>
             </div>
```
Comment (roshan-bajgain) [inline index.html:178]: - [ ] Kindly make this about section as similar to Figma design
### Your design
his
### Figma design
figma

**Review by roshan-bajgain — CHANGES_REQUESTED** [review]
Hi @AsimKhan2019, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️
- Professional README file. ✔️
- PR has a good title and summary. ✔️

## Required Changes ♻️

- [x] Kindly follow the button styling: Active, Hover, and Pressed (see design). Implement in your all buttons
### Figma design

hisssssssss

- [ ] I cannot make a comment there so I am giving here kindly cover each content with a `section tag` not the div tag for more semantic HTML.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (AsimKhan2019) [reply index.html]: I have made some changes, just like you asked. Please review and let me know.

**Comment by AsimKhan2019** [comment]
Made the changes accordingly.

File: style.css [hunk 2]
```diff
@@ -31,6 +31,21 @@ button.primary-btn {
...
+  background-color: #6070ff;
+  color: #fff;
+}
+
+button.primary-btn:active {
+  background-color: blue;
+  color: #fff;
+}
+
+#container {
```
Comment (Ismailco) [inline style.css:44]: - [ ] [OPTIONAL] The main container of the projects has a ```display:grid``` kindly implement this it's a requirement

File: style.css [hunk 3]
```diff
@@ -94,6 +109,18 @@ h1 {
...
+  margin-top: 10px;
+  height: 67px;
+  display: flex;
+  align-items: center;
+  padding: 11px;
+  font-weight: 500;
+  color: #253858;
+}
+
 .work-card .skills li {
```
Comment (Ismailco) [inline style.css:124]: - [ ] [OPTIONAL] The content of the project is aligned with the flexbox, please use flexbox to align the projects here

File: style.css [hunk 4]
```diff
@@ -154,13 +201,35 @@ h1 {
...
+  height: 50px;
+  margin-right: 15px;
+}
+
 .canopy li.item span {
   margin-left: 5px;
   color: #7a869a;
 }
 
 #about {
```
Comment (Ismailco) [inline style.css:216]: - [ ] The content of ```About``` section should be aligned with flexbox, please implement this it's a requirement

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @AsimKhan2019,

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights

✔️  No linter errors
✔️ Used Github Flow
✔️ Work documented professionally

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 2 above)
Comment (AsimKhan2019) [reply style.css:44]: Can you please elaborate a little further about this requirement., I do not quite understand it?

File: style.css (hunk 3 above)
Comment (AsimKhan2019) [reply style.css:124]: Can you please elaborate a little further about this requirement, I do not quite understand it?

File: style.css (hunk 4 above)
Comment (AsimKhan2019) [reply style.css:216]: Can you please elaborate a little further about this requirement, I do not quite understand it?

Comment (Ismailco) [reply style.css:216]: @AsimKhan2019, you need to use flexbox to align the whole section. to use flexbox ```display: flex```

File: style.css (hunk 2 above)
Comment (AsimKhan2019) [reply style.css:44]: @Ismailco should I add the display: grid to the container element?

File: style.css (hunk 4 above)
Comment (AsimKhan2019) [reply style.css:216]: @Ismailco I used display: flexbox to the about section, is that right?

File: style.css (hunk 3 above)
Comment (AsimKhan2019) [reply style.css:124]: @Ismailco should I add the display: flex in  .work-card element?

**Comment by AsimKhan2019** [comment]
I have implemented all the fixes and optional requirements to the best of my knowledge. I require more understanding of things that will take time.

File: style.css (hunk 4 above)
Comment (AsimKhan2019) [reply style.css:225]: I implemented whatever I could understand from the comment.

File: style.css (hunk 2 above)
Comment (AsimKhan2019) [reply style.css:44]: I implemented whatever I could understand from the comment.

**Comment by AsimKhan2019** [comment]
Updated style.css
Added display : grid to #container

**Review by jaflih — APPROVED** [review]
Hi @AsimKhan2019   , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 




## Highlights
No linter errors. ✔️
GitHub flow is followed. ✔️
Professional README file. ✔️
PR has a good title and summary. ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**